tools/sigh.ts

```diff
@@ -318,17 +323,37 @@ function railroad(): boolean {
   return true;
 }
 
-function buildPath(path: string, deps?: string[]): () => boolean {
+function cleanObsolete() {
```

Comment:
A comment for this function would be nice

Reply:
Done.